$NetBSD: patch-Configurations_10-main.conf,v 1.1 2020/07/22 20:41:30 sjmulder Exp $

Add support for Apple Silicon. Imported from open pull request:
https://github.com/openssl/openssl/pull/12369

--- Configurations/10-main.conf.orig	2020-04-21 12:22:39.000000000 +0000
+++ Configurations/10-main.conf
@@ -1557,6 +1557,14 @@ my %targets = (
         bn_ops           => "SIXTY_FOUR_BIT_LONG",
         perlasm_scheme   => "macosx",
     },
+    "darwin64-arm64-cc" => {
+        inherit_from     => [ "darwin-common", asm("aarch64_asm") ],
+        CFLAGS           => add("-Wall"),
+        cflags           => add("-arch arm64"),
+        lib_cppflags     => add("-DL_ENDIAN"),
+        bn_ops           => "SIXTY_FOUR_BIT_LONG",
+        perlasm_scheme   => "ios64",
+    },
 
 ##### GNU Hurd
     "hurd-x86" => {
